csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -409,6 +409,13 @@ private void TestTensorRTProviderOptions()
                 skipModels["coreml_VGG16_ImageNet"] = "System out of memory";
                 skipModels["test_ssd"] = "System out of memory";
                 skipModels["roberta_sequence_classification"] = "System out of memory";
+                // models from model zoo
+                skipModels["VGG 19"] = "bad allocation";
```

Comment:
We know in 32-bit mode we don't have enough memory to run this model. In 64-bit mode it's fine.